### ISSUE TITLE
Request to package your code in a portable way

### ISSUE THREAD

**Issue by Derpitron** [fuzzydragon/Roblox-Game-Presence-For-Discord#2]
I have used your code in a project of mine, https://github.com/Derpitron/roblox-experience-log. It gets the current played game on roblox then posts it's game link to a Discord webhook.

I appreciate your code and the rich presence works great for me, but I feel that if the code was packaged in some way people could use it more easily for functions other than Rich Presence. I don't know how exactly to say it, do you have a separate contact aside from Github?

**Comment by fuzzydragon** [comment]
Sorry for getting so late back to you. You can contact my email directly at andrewj@cock.email

I'm not sure what you mean. This is the first Go program I've ever written. I built the releases awhile ago under the old Go package manager so I'm not sure how this scales with the new one. 

Can you clarify what you mean by "but I feel that if the code was packaged in some way people could use it more easily for functions other than Rich Presence"

Are you wanting a go module so you can require my repository and access methods like "GetProcessByName" or "UpdateRobloxPresence"?

**Comment by Derpitron** [comment]
> Are you wanting a go module so you can require my repository and access methods like "GetProcessByName" or "UpdateRobloxPresence"?

Yes, that is what I mean! My program that I made a few weeks back takes the code of GetProcessByName and UpdateRobloxPresence directly and uses it, but It would be more convenient to simply import the library and use the methods, and it would be more ethical as I do not want it to appear as if I have stolen your code

**Comment by fuzzydragon** [comment]
I will do this in a week or so. 

I most likely will end up creating two new repositories: one for the _Roblox specific methods_ and one for the _process grabbing related methods_. 

This way you will be able to do something like: 

```go
import (
   "github.com/fuzzydragon/process-util"
   "github.com/fuzzydragon/roblox-presence-util"
)
```

and then maybe

```go
roblox := processutil.GetProcessByName("ROBLOX")
```

**Comment by fuzzydragon** [comment]
Busy with work. I will try to get this done before the end of September.

**Comment by Derpitron** [comment]
Take your time friend
